setup.py

```diff
@@ -9,7 +9,7 @@
 SOURCE_DIR = os.path.join(ROOT_DIR)
 
 requirements = [
-    'requests >= 2.5.2',
+    'requests >= 2.5.2, < 2.11',
```

Comment:
You can use any version >= 2.5.2, except 2.11.0. 
2.11.1 fixed the bug so that version or above is fine as well.